_opensearch/rest-api/index-apis/put-mapping.md

```diff
@@ -36,23 +39,29 @@ PUT /<target-index>/_mapping
 PUT /_mapping
 ```
 
+You can also use the put mapping operation to update multiple indexes with one request.
+
+```
+PUT /<target-index1>,<target-index2>/_mapping
+```
+
 ## URL parameters
 
-All put mapping parameters are optional.
+All update mapping parameters are optional.
```

Comment:
Using "put mapping" here might be more accurate, since that's HTTP request we're using